omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -169,19 +169,26 @@
 default['supermarket']['nginx']['default']['modules'] = []
 
 # ## Postgres
-
 default['supermarket']['postgresql']['enable'] = true
+default['supermarket']['postgresql']['external'] = false
+default['supermarket']['postgresql']['version'] = '13'
 default['supermarket']['postgresql']['username'] = node['supermarket']['user']
-default['supermarket']['postgresql']['data_directory'] = "#{node['supermarket']['var_directory']}/postgresql/9.3/data"
+default['supermarket']['postgresql']['data_directory'] = "#{node['supermarket']['var_directory']}/postgresql/#{node['supermarket']['postgresql']['version']}/data"
 
 # ### Logs
 default['supermarket']['postgresql']['log_directory'] = "#{node['supermarket']['log_directory']}/postgresql"
 default['supermarket']['postgresql']['log_rotation']['file_maxbytes'] = 104857600
 default['supermarket']['postgresql']['log_rotation']['num_to_keep'] = 10
 
 # ### DB settings
+# The following 2 attributes are introduced for upgrading postgres to v13.3
+default['supermarket']['postgresql']['max_wal_size'] = '1GB'
+default['supermarket']['postgresql']['min_wal_size'] = '80MB'
+
+default['supermarket']['postgresql']['checkpoint_flush_after'] = '256kB'
 default['supermarket']['postgresql']['checkpoint_completion_target'] = 0.5
-default['supermarket']['postgresql']['checkpoint_segments'] = 3
+# This attribute has been deprecated after postgres 9.3
+# default['supermarket']['postgresql']['checkpoint_segments'] = 3
```

Comment:
Have we updated the docs for these attributes to mark them as deprecated?

Reply:
no we haven't

Reply:
Have added it now.

Reply:
<img width="780" alt="Screenshot 2022-01-11 at 11 23 30 AM" src="https://user-images.githubusercontent.com/85236476/148888762-6d22b3c8-7d0a-4c32-8945-ee0e91b06230.png">
@tas50 this is how it will look.